banzai_nres/rv.py

```diff
@@ -49,7 +46,7 @@ def cross_correlate(velocities, wavelength, flux, flux_uncertainty, template_wav
     variance = flux_uncertainty * flux_uncertainty
 
     for v in velocities:
-        doppler = 1.0 / (1.0 + v / c)
+        doppler = 1.0 * units.dimensionless_unscaled / (1.0 * units.dimensionless_unscaled + v / constants.c)
```

Comment:
I have had issues in the past with astropy units massively slowing down computation time in loops. Just want to verify that this does not slow down the RV computation.